shell/test/get_max_or_min_int.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+. ../common/common_benchmarks.sh
+
+big_number="10"
+small_number="5"
+
+@test "get_max_int should return the higher natural number" {
+    run get_max_int "$big_number" "$small_number"
+    [ "$status" -eq 0 ]
+    [ "$output" = "$big_number" ]
+
+    run get_max_int "$small_number" "$big_number"
+    [ "$status" -eq 0 ]
+    [ "$output" = "$big_number" ]
+}
+
+@test "get_min_int should return the smaller natural number" {
+    run get_min_int "$big_number" "$small_number"
+    [ "$status" -eq 0 ]
+    [ "$output" = "$small_number" ]
+
+    run get_min_int "$small_number" "$big_number"
+    [ "$status" -eq 0 ]
+    [ "$output" = "$small_number" ]
+}
+
+@test "both get_[max|min]_int fail if only one parameter is passed" {
+    run get_min_int "$big_number" ""
+    [ "$output" = '../common/common_benchmarks.sh: línea 249: ((: 10 <  : error sintáctico: se esperaba un operando (el elemento de error es "<  ")' ]
```

Comment:
Checking for the full error message with line number and in Spanish means that this test will only pass in very specific conditions.

It would be better if the function checked its arguments and exited with a non-zero status if the number of arguments or the type were incorrect. Then here you would just test that `$status -ne 0`